### PR TITLE
feat: NFT Collections: add 24-hour price change and delta arrow

### PR DESCRIPTION
**Pull request by gbugyis**
Adds 24-hour floor price change to Collection Stats (to match Figma)

### CODE REVIEW

File: src/nft/components/collection/CollectionStats.tsx [hunk 1]
```diff
@@ -254,6 +263,9 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
   // round daily volume & floorPrice to 3 decimals or less
   const totalVolumeStr = ethNumberStandardFormatter(stats.stats?.total_volume)
   const floorPriceStr = ethNumberStandardFormatter(stats.floorPrice)
+  const floorChangeStr =
+    stats.stats && stats.stats.one_day_change ? Math.round(Math.abs(stats.stats.one_day_change) * 100) : 0
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:267]: should be `stats.stats.one_day_floor_change` not `one_day_change`. One day change refers to 1 day TVL % change

File: src/nft/components/collection/CollectionStats.tsx [hunk 2]
```diff
@@ -10,10 +11,18 @@ import { ethNumberStandardFormatter } from 'nft/utils/currency'
...
+import styled from 'styled-components/macro'
 
 import { DiscordIcon, EllipsisIcon, ExternalIcon, InstagramIcon, TwitterIcon, VerifiedIcon, XMarkIcon } from '../icons'
 import * as styles from './CollectionStats.css'
 
+const PercentChange = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  justify-content: center;
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:23]: two `justify-content`s, the overwritten one can be removed

File: src/nft/components/collection/CollectionStats.tsx [hunk 3]
```diff
@@ -282,6 +294,13 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
           {floorPriceStr} ETH
         </StatsItem>
       ) : null}
+      {stats.stats?.one_day_change ? (
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:297]: `stats.stats.one_day_floor_change`

File: src/nft/components/collection/CollectionStats.tsx (hunk 1 above)
Comment (gbugyis) [reply src/nft/components/collection/CollectionStats.tsx]: changed it, but atm that field is missing from the response. Need to follow up with BE

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**